packages/config/config/devices/0x0234/templates/logic_group_template.json

```diff
@@ -564,9 +557,9 @@
 		"allowManualEntry": true
 	},
 	"watt_threshold_power": {
-		"label": "Watt Threshold: Power Off",
-		"unit": "watt",
+		"label": "Power Change Threshold: Power Off",
```

Comment:
This refers to the total power, not the power change, does it?

Reply:
This refers to the total power output of the socket
I guess it should be called upper or upper limit or so

Reply:
Maybe just `Upper Power Limit`?